schema/data_feed/tracker.fbs

```diff
@@ -9,34 +9,50 @@ namespace solarxr_protocol.data_feed.tracker;
 /// Trackers may be synthetic/computed or instead part of an actual hardware device.
 /// There can be multiple trackers per hardware device.
 table TrackerData {
-   	tracker_id: solarxr_protocol.datatypes.TrackerId;
+    tracker_id: solarxr_protocol.datatypes.TrackerId;
 
-	info: TrackerInfo;
+    info: TrackerInfo;
 
-	status: solarxr_protocol.datatypes.TrackerStatus;
+    status: solarxr_protocol.datatypes.TrackerStatus;
 
-	// Everything here is optional and controlled by `TrackerDataMask`
-	rotation: solarxr_protocol.datatypes.math.Quat;
-	/// Position, in meters
-	// TODO: With respect to what space?
-	position: solarxr_protocol.datatypes.math.Vec3f;
-	/// Raw rotational velocity, in euler angles
-	raw_rot_vel: solarxr_protocol.datatypes.math.Vec3f;
-	/// Raw translational acceleration, in m/s^2
-	raw_trans_accel: solarxr_protocol.datatypes.math.Vec3f; // trans? BASED
-	/// Temperature in degrees celsius
-	temp: solarxr_protocol.datatypes.Temperature;
+    // Everything here is optional and controlled by `TrackerDataMask`
+
+    /// Raw sensor rotation
```

Comment:
```suggestion

    /// Sensor rotation after fusion 
```

Reply:
The goal is to be general. That would exclude Vive trackers, and anything that isn’t pure IMU.
Maybe saying “Raw tracker rotation” (instead of sensor) would be better?